components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthComponentServiceHolder.java

```diff
@@ -243,4 +245,34 @@ public void setOrganizationUserResidentResolverService(
 
         this.organizationUserResidentResolverService = organizationUserResidentResolverService;
     }
+
+    /**
+     * Register access token response handlers for modifying token response attributes.
+     *
+     * @param accessTokenResponseHandler {@link AccessTokenResponseHandler} instance
```

Comment:
Missing full stop at the end of the comment. Please fix this in other places as well.